WIP: issue-113 Allowed to xref within a module? (#162)
redhat-documentation/modular-docs: modular-docs-manual/files/TEMPLATE_CONCEPT_concept-explanation.adoc

////
Optional. Delete if not used.
////

> * A bulleted list of links or xrefs to other material closely related to the contents of the concept module.

Reviewer: Is this necessary? Xrefs are just another kind of a link. As we're not calling out the specific AsciiDoc macros (`link:`) -- we're just talking about links -- then I see no point in adding "xrefs".

(Same for the other two templates.)

Not to mention that using "xrefs" as a word is really jargony :)

Reviewer 2: FWIW, I prefer the specific inclusion of xref.  URLs (using the link macro[1]), to content such as a KCS, were not forbidden.  An xref to another file within the same repo _was_ forbidden.   There's a suggestion to prefer the xref macro to add a link to a relative AsciiDoc file[2].

[1] https://docs.asciidoctor.org/asciidoc/latest/macros/link-macro/
[2] Same as above, but scroll to the TIP under the "Link to a relative file" heading.

Reviewer 3: > Is this necessary? Xrefs are just another kind of a link. As we're not calling out the specific AsciiDoc macros (`link:`) -- we're just talking about links -- then I see no point in adding "xrefs".
> 
> (Same for the other two templates.)
> 
> Not to mention that using "xrefs" as a word is really jargony :)

I think mentioning cross-references is helpful. Otherwise, some writers reading this content might interpret the phrase as excluding the `xref:` macro and continuing "the ban on xrefs." 

To help folks who aren't familiar with xrefs, we could apply the proposed change, "...bulleted list of links to other material closely," and append something like "These links can include `link:` and `xref:` macros."

Reviewer 3: With Preeti and Robert's thumbs up, I'll suggest a change to this line in the docs.

> * A bulleted list of links or xrefs to other material closely related to the contents of the concept module.

Reviewer 3: ```suggestion
* A bulleted list of `link:` and `xref:` links to closely-related topics.
```

> * A bulleted list of links or xrefs to other material closely related to the contents of the concept module.

Reviewer 3: ```suggestion
* A bulleted list of links to other closely-related material. These links can include `link:` and `xref:` macros.
```
